docs/standalone-installation.md

```
@@ -35,16 +94,19 @@ This will:
+> If it isn't, add `export PATH="$HOME/bin:$PATH"` to your shell profile
+> (e.g., `~/.zshrc` or `~/.bashrc`).
 
-> If setup.sh identifies conflicting syntax highlighting extensions, it will ask you what to do (disable/uninstall/do nothing).
+> If setup.sh identifies conflicting syntax highlighting extensions, it will
+> ask you what to do (disable/uninstall/do nothing).
 
 Requires [Bun](https://bun.sh) (`brew install bun` or see https://bun.sh).
 
 ## CLI Options
 
 ```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)